### ISSUE TITLE
"Standard" Reveal width doesn't match docs (Foundation 6)

### ISSUE THREAD

**Issue by pdeffendol** [zurb/foundation-sites#8707]
**How can we reproduce this bug?**

1. Build a Reveal modal using Foundation 6
2. View on a medium or larger screen width
3. See fixed 600px dialog

**What did you expect to happen?**

According to http://foundation.zurb.com/sites/docs/reveal.html#sizing the width of a medium+ Reveal dialog should be 80%.  This is what I expected coming from Foundation 5 as well.

**What happened instead?**

The width is set to 600px.

Either the documentation or the code is wrong.  I'm inclined to say it's the code that needs to be fixed.

**Comment by designerno1** [comment]
in v6 there is currently no `medium` Reveal, so it falls back to the default width defined in settings
```scss
$reveal-width: 600px !default;
$reveal-max-width: $global-width !default;
```
docs
```html
.tiny: 30% wide
.small: 50% wide
.large: 90% wide
.full: 100% width and height, defaults the escClose option to true, as well as creates a close button.
```
the docs say `On medium-sized screens and larger, the width changes to 80%.` thats a little bit confusing

**Comment by brettsmason** [comment]
I've just looked into this a bit more, and its caused by this in `_reveal.scss`
```
@include breakpoint($-zf-zero-breakpoint only) {
    @include reveal-modal-fullscreen;
}
```
So the smallest breakpoint is always receiving the `.full` styling. Should this be the case or should the selected classes height be at least respected and maybe have a small gap left/right? I'd be interested to see what others opinions are on this @zurb/yetinauts

**Comment by pdeffendol** [comment]
@brettsmason I was more expecting the default width of the Reveal to be a percentage, which is consistent with the named widths (.small, etc.).

**Comment by gremmie** [comment]
I ran into this also. Seems like a bug. The default width is not 80% as advertised in the docs. The other sizes appear to work as documented.

**Comment by DanielRuf** [comment]
What value should be used here? Is the documentation or the code right?

**Comment by gormus** [comment]
I use a much narrow page layout width on my project:  `$global-width: rem-calc(540);`

And no matter what size of reveal modal I call they all are restricted with reveal's `max-width` value. 

As @designerno1 mentioned in their comment above, reveal has this restricting rule: `$reveal-max-width: $global-width !default;`

The question is whether the reveal modal's maximum width should be restricted by `$global-width` or browser window width. I'd expected the later.

**Comment by DanielRuf** [comment]
> $reveal-max-width

You can generally overwrite this in your settings.

**Comment by DanielRuf** [comment]
A modal width of 100vh would make it wider than the max page width which should not happen.

**Comment by gormus** [comment]
@DanielRuf I agree `$reveal-max-width` can be overwritten in the settings. And that's what I'm doing too. I just thought it might be better to provide much accommodating defaults out-of-box.

Modal widths are already defined via `.tiny`, `.small`, and `.large` classes. So I was thinking simply removing the `max-width` definition, not necessarily setting it to 100vw actually.
